src/assets/icons/june.svg

```diff
@@ -1,7 +1,9 @@
-<path d="M38.925 23.433c-0.687-0.209-9.821-3.582-24.896 0.030-1.493 0.358-1.821 0.806-1.582 1.672 0.507 1.761 8.418-2.985 25.761 1.343 0.896 0.209 2.060-0.149 2.328-1.045 0.239-0.836 0-1.522-1.612-2z"></path>
-</svg>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="145px" height="90px" viewBox="0 0 145 90" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
+    <title>Slice 1</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
Is dit het kleurrijke logo voor op mobiel? Best een betere naam geven.

Reply:
Nee dit is niet dat Logo, is het gewone June logo, maar staat dubbel dus er mag 1 weg. Ik zal dat even doen